app/views/security/_mfa_token.html.erb

```diff
@@ -10,7 +10,7 @@
       </span>
       <span class="govuk-!-font-weight-bold">
         <%= humanized_date(mfa_token.created_at) %>
-        to
+        <%= t("account.security.security_codes.code_description.conjunction") %>
```

Comment:
Mmm I always find these ones really difficult to work out what to do.
usually we've got a sentence in a locale right? So we can assume that where we had an english sentence we might have the same in another language.

Things like this assume a grammatical format tough no?
So if we translated this into a locale that had a different use of conjunctions (or no conjunctions) this would then break?

That said we don't actually have any translations in mind, so not an issue here.
But i wonder if there's a better way that keeps the ease of use but allows these locale files to be used for their initial purpose too?

Perhaps if these were passed as variables?
I suspect it would need a more complicated helper with flow control.. 🤔

Reply:
Much better than hard coded though!

Reply:
I think we'd need to pass the start and end date as variables, yeah.  And we might need to think about different date formats for different locales (not just translating the month name) too...